invenio_records_lom/services/components.py

```diff
@@ -26,37 +26,50 @@ class ResourceTypeComponent(ServiceComponent):
 
     new_version_skip_fields = ["publication_date", "version"]
 
-    # pylint: disable-next=unused-argument
     def create(
-        self, identity: Identity, data: dict = None, record: Record = None, **kwargs
+        self,
+        identity: Identity,
+        data: dict = None,
+        record: Record = None,
+        errors=None,  # pylint: disable=unused-argument
+        **kwargs,
```

Comment:
`**kwargs` was left in here, but removed for all the other methods

Is the different treatment of `**kwargs`-removal intentional?

Reply:
yes it is intentionally there, because otherwise following would pop up:
`W: 29, 4: Variadics removed in overridden 'ResourceTypeComponent.create' method (arguments-differ)`